lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -33,6 +33,7 @@ export async function createNewSession(
     tenantId: string,
     recipeUserId: RecipeUserId,
     disableAntiCsrf: boolean,
+    userContext: any,
```

Comment:
should be the last param, and non optional.. please check in all places! So you can remove the `= {}` from the other two params of this function, and manually check all places where this function is being called to make sure that the params are correctly provided to the function call.